src/redemptions/redemption.service.ts

```diff
@@ -296,7 +301,16 @@ export class RedemptionService {
         age,
       };
     }
-
+    const matchedApproval = matchApprovedLabels(transactionStatus);
+    if (matchedApproval) {
+      return {
+        status: KycStatus.SUCCESS,
+        failureUrl: null,
+        failureMessage: `Approved benign label set: ${matchedApproval}`,
```

Comment:
Failure message is for failures, not success reasons.

```suggestion
        failureMessage: ``,
```

Reply:
Where else can we record the reason for success? It will be useful to have this for debugging